### ISSUE TITLE
Why don't you add the regularization loss here ?

### ISSUE THREAD

**Issue by shamanez** [aymericdamien/TensorFlow-Examples#179]
In this [line of the convolutional neural network](https://github.com/aymericdamien/TensorFlow-Examples/blob/master/examples/3_NeuralNetworks/convolutional_network.py#L87)
